lib/modules/platform/scm-manager/schema.ts

```diff
@@ -0,0 +1,122 @@
+import { z } from 'zod';
+
+export const UserSchema = z.object({
+  mail: z.string().optional().nullable(),
+  displayName: z.string(),
+  name: z.string(),
+});
+export type User = z.infer<typeof UserSchema>;
```

Comment:
```suggestion
export type User = z.infer<typeof User>;
```
typescript is smart enough to distinguish between type and Schema, so same name is fine

Reply:
I have adopted your change, since zT-1337 isn't available right now. I hope I've got it right :-)